app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -970,6 +970,13 @@ class SmokeTest {
             verifyFolderTitle("My Folder")
             IdlingRegistry.getInstance().unregister(bookmarksListIdlingResource!!)
         }.openThreeDotMenu("Test_Page_1") {
+        }.clickEdit {
+            clickParentFolderSelector()
+            selectFolder("My Folder")
+            navigateUp()
+            saveEditBookmark()
+            createFolder("My Folder 2")

```

Comment:
Do we need to register any idling resource here like you did above?

Reply:
Yes, it makes sense, overlooked this.